lib/src/metta/runner/mod.rs

```diff
@@ -241,11 +241,43 @@ impl Metta {
         self.0.settings.borrow().get(key.into()).map(|a| a.to_string())
     }
 
+    /// Runs a MeTTa program expressed as a string of characters
+    pub fn run_program_str(&self, program: &str) -> Result<Vec<Vec<Atom>>, String> {
```

Comment:
I would not add this method, it is trivially replaced by `metta.run(SExprParser::new(program))`

Reply:
Ok.  I thought it would be a nice convenience to give users a 1-line method to execute code.  But I see your desire to have syntax-independence at the Runner level is more important.

Removed in d26b225